iphone/cli/commands/_build.js

```diff
@@ -45,7 +45,7 @@ var appc = require('node-appc'),
 	series = appc.async.series,
 	version = appc.version;
 var platformsRegExp = /^(android|ios|iphone|ipad|mobileweb|blackberry|windows|tizen)$/;
-const pemCertRegExp = /(^-----BEGIN CERTIFICATE-----)|(-----END CERTIFICATE-----.*$)|\n/g;
+var pemCertRegExp = /(^-----BEGIN CERTIFICATE-----)|(-----END CERTIFICATE-----.*$)|\n/g;
```

Comment:
The usage here is fine and I had left it in my previous PR, but decided to remove this time to keep the file cleaner.